style/question.less

```diff
@@ -86,40 +104,115 @@
 
 
   // &.mode-preview {
-    .answer-input-box { display: table-cell; }
 
-&:not(.has-correct-answer) {
+    .answers-table {
+      .answers-answer {
+        counter-increment: answer 1;
+        display: block;
+        width: 100%;
+        clear: both;
+
+        .answer-letter {
+          display: inline-block;
+          width: 2rem;
+          font-weight: bold;
+          &:after { content: counter(answer, lower-latin) ')'; }
+        }
+
+        .answer-content {
+          display: inline-block;
+          padding: 1rem;
+          padding-right: 0;
+        }
+      }
+    }
 
-    .answer-input-box:checked + label.answer-label,
-    .answer-input-box:checked + label.answer-label:hover {
-      background-color: #9e9e9e; // darker than hover
+    .answer-input-box {
+      display: inline-block;
+      float: left;
+      font-size: 1.5em;
+      margin: 0.95em 0;
+      margin-left: -30px;
     }
 
-}
+    .answers-table label.answer-label {
 
-    .answers-table {
+      margin: 0.5em 0;
+      padding-left: 0;
+      line-height: 1.5em;
+      display: inline-block;
       width: 100%;
+      float:left;
+      transition: background-color .2s ease-in;
+      transition: padding-left .2s ease-in;
+
+      //Materialize hack
```

Comment:
I'm using the transitions, the Materialize hacks, I haven't tried taking them out, but I will to see if it makes a difference.